app/controllers/api/account.php

```diff
@@ -2467,7 +2468,12 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
                 $abuse = new Abuse($timelimit);
                 if ($abuse->check() && System::getEnv('_APP_OPTIONS_ABUSE', 'enabled') === 'enabled') {
                     $helper = PhoneNumberUtil::getInstance();
-                    $countryCode = $helper->parse($phone)->getCountryCode();
+
+                    try {
+                        $countryCode = $helper->parse($phone)->getCountryCode();
```

Comment:
Can we try and understand what inputs lead to this exception?

This should then be handled in the Phone() validator and we should have tests for these cases